_data/pascaladhikary/pascaladhikary.md

```diff
@@ -0,0 +1,7 @@
+---
+name: pascaladhikary
+institution: Illinois Mathematics & Science Academy
+profile_pic: mona.png
```

Comment:
Image file path name seems incorrect, please change it from `mona.png` to `pascaladhikary.png`